package.json

```diff
@@ -23,6 +23,7 @@
     "demo:build": "nwb build-react-component src/index.js"
   },
   "dependencies": {
+    "antd": "^3.4.1",
```

Comment:
Could this potentially clash with the dependencies of the components that's using this? This is a general question, not only about antd, I just know that specific one is in fact in operational-intelligence as well.

Reply:
I don't know anything about declaring dependencies in NodeJS. Should I remove it and use a different component instead?

Reply:
Just leave it, I'll look into it.